src/Microsoft.AspNet.OData.Shared/ODataQueryParameterBindingAttribute.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using Microsoft.AspNet.OData.Query;
```

Comment:
i meant, the using statement. doesn't seem needed.

---
In reply to: [139828269](https://github.com/OData/WebApi/pull/1075#discussion_r139828269) [](ancestors = 139828269)

Reply:
It's because of <see cref="ODataQueryOptions"/> in the comments.

---
In reply to: [139848296](https://github.com/OData/WebApi/pull/1075#discussion_r139848296) [](ancestors = 139848296,139828269)